data/json/items/comestibles/med.json

```diff
@@ -134,9 +134,97 @@
     "use_action": {
       "type": "consume_drug",
       "activation_message": "You take some aspirin.",
-      "effects": [ { "id": "pkill1", "duration": 720 } ]
+      "effects": [ { "id": "pkill1_nsaid", "duration": 720 } ]
```

Comment:
I think all these durations probably need to be looked at. This is 12 minutes, I doubt it's been checked or thought in years. Ideally, the effect should ramp up after half an hour or so and last for around 3-4 hours, tapering off towards the end. The ramp up I know may be challenging and you don't need to do that, but maybe we should have it last longer at least. Same goes for tylenol (4-6 hours) and ibuprofen (3-4 hours).

I realize making it taper off at the end would require adjusting the levels of the effect and some other junk. It's not necessary for this PR.

Reply:
So, the vitamin system I have would in principle allow for both ramping up and tapering off of effects. Currently, I have simplified it so that the ramp-up stage is instant, in order to make balance and duration calculations possible, but this need not be the case in the future. The tapering off is easily possible by just making the effects even more granular, but honestly, since the pkill system in general needs to be overhauled, this should probably be avoided for this PR.

Reply:
Ok, I will increase the duration to around 2.5 hours instead of 12 minutes. This is a bandaid fix until we rationalize pkill using vitamins or other resources.